QATCH/VisQAI/src/models/predictor.py

```
@@ -306,24 +445,51 @@ def predict_uncertainty(
+            # Calculate coefficient of variation
+            cv = np.where(mean_preds != 0,
+                          result['std'] / np.abs(mean_preds), 0)
+
+            uncertainty = {
+                'std': result['std'],
+                'lower_ci': result['lower_ci'],
+                'upper_ci': result['upper_ci'],
+                'coefficient_of_variation': cv
+            }
+
+            return mean_preds, uncertainty
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)